### ISSUE TITLE
Wrong implementation of skipto plugin on the joomla.org

### ISSUE THREAD

**Issue by zwiastunsw** [joomla/joomla-websites#2015]
1. The SkipTo script on the Joomla.org homepage registers and displays links to all header and footer elements on the page. These are not the elements the aplugin should be displaying links to. Only the link to the page page header and footer page should be displayed in SkipTo. Headers and footers of elements other than the body element are not landmarks.

To correct this error, you can:
- add role attributes to the header (page) and footer (page) tags: `<header role="banner" class="header">` and `<footer role="conteninfo"  class="footer text-center">`; in the script settings you must add: `'landmarks: 'main, [role="banner"], [role="conteninfo"]...'`,
- or remove the header and footer tags in the ‘Download Joomla’, ‘Use Joomla’, ‘Learn Joomla’ modules. 

2. Links to nav elements do not have unique accessible names. If there is more than one landmark of a given type on a page, each must have a unique accessible name

To fix this error, add the aria-label attribute to nav elements, for example:
`<nav class="navigation" role="navigation" aria-label="main">`
`<nav class="subnav-wrapper" aria-label="secondary">`
`<nav class="navbar navbar-expand" aria-label="secondary">`
In footer:
`<nav class="navbar navbar-expand" aria-label="socialmedia">`
`<nav class="navbar navbar-expand" aria-label="quick links">`

![joomla-org](https://github.com/user-attachments/assets/97b7d9e4-4296-416e-8546-2fc04b30d72a)

**Comment by wilsonge** [comment]
The nav part of this issue has been fixed in the base template, there's one last fix to make before we roll it out. I'm expecting a rollout to the sites over the next 1-2 weeks.

Keeping this issue open because the template doesn't affect the markup on the joomla.org site itself for the 1st part.

**Comment by wilsonge** [comment]
Nav part now rolled out to all j4 sites
